docs/source/jupyter.rst

```diff
@@ -16,3 +16,4 @@ Jupyter notebooks
    /examples/Using NEOS package.ipynb
    /examples/Visualizing the SpaceX Tesla Roadster trip to Mars.ipynb
    /examples/Natural and artificial perturbations.ipynb
+   /examples/styling.ipynb
```

Comment:
Can we please rename this one to something more expressive, like `Customizing orbit plots.ipynb`? (Also, first letter in caps!)